### DISCUSSION TITLE
Coupling LIS with WRF

### DISCUSSION THREAD

**Discussion by shaider2** [NASA-LIS/LISF#1573] [Ideas]
Hi everyone, I a looking for some suggestions or papers which describes the steps or methodologies for coupling LIS with WRF

**Comment by dmocko** [comment]
Hi @shaider2 

You are in luck.  LIS has already been coupled to WRF - via the NASA Unified-WRF project:
    https://nuwrf.gsfc.nasa.gov/

Please see the website to see how to get the software.

**Comment by shaider2** [comment]
Hi @dmocko 
Thank you for the response. For my LISF simulation I am using Noah LSM v 3.3.
I am using WRF 4.5.2 for the seasonal simulation. Unfortunately I am unable to figure out the version of Noah lsf is WRF. I just wanted to make sure that I am using the same version for both model.
